magento/magento2
# Allow to load base64 encoded images and fonts

### Description
Current Content Security Policy setup doesn't allow to load images and fonts via `data:` scheme which is quite a popular method to load local resources (base64 encoded images). Additionally, it's used internally by the browser cache (Chrome) when you refresh the page.

### Manual testing scenarios (*)
1. Open Luma homepage and open browser console.
2. Refresh the page to trigger resource loading from the browser cache.
3. Inspect errors in the console:
  <img width="628" alt="Font errors in console" src="https://user-images.githubusercontent.com/306080/80577266-441b3e00-8a0f-11ea-8156-87f043503fd1.png">
4. Apply the patch, and try again. No errors should appear in the log.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31170: Allow to load base64 encoded images and fonts
